.lockfiles/py310-dev.lock

```diff
@@ -842,6 +842,7 @@ tbb==2021.13.0 ; platform_system == 'Windows'
     # via mkl
 tenacity==8.5.0
     # via
+    #   baybe (pyproject.toml)
```

Comment:
Doc seems to fail due to an error in the MAB description, could you verify? Seems to be an issue with some backtick 
<img width="1444" alt="image" src="https://github.com/user-attachments/assets/3cca03e8-68cc-44aa-9774-dfa01123b0e0">

Reply:
Looking at the changed example, I'd guess that it is due to empty lines that were removed?

Reply:
solved thx